packages/card/primitives.tsx

```diff
@@ -5,7 +5,7 @@
 
 import * as React from 'react'
 import classNames from 'classnames'
-import Image from 'next/image'
```

Comment:
I think `<Card>` actually needs `next/image` since @alexcarpenter fixed the CSS.

Reply:
Ah, great catch! I will revert this then 👍